src/components/ProductCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Card, CardBody, CardFooter, Image } from '@nextui-org/react';
+
+const ProductCard = ({ product }) => {
+  return (
+    <Card shadow='sm' isPressable className='border-2 w-[300px] h-[450px] flex-shrink-0'>
```

Comment:
We need to wrap this with `Link` component from `react-router-dom` to enable navigation to `Product` page. We can do like

```javascript
import { Link } from 'react-router-dom';


return (<Link to={`/product/${product.slug}` alt={`link-to-${product.name}`}}>{....}</Link>
```